doc/code/targets/prompt_targets.py

```diff
@@ -46,11 +45,50 @@
 with AzureOpenAIChatTarget() as azure_openai_chat_target:
     print(azure_openai_chat_target.send_prompt(prompt_request=request))
 
+# %% [markdown]
+# In this demo section, we illustrate instances of the multimodal inputs for AzureOpenAIChatTarget. GPT-V deployment is being utilized within the Azure OpenAI resource.
+# 
+# _Note:_ to run this section of the demo you need to setup your `.env` file to properly authenticate to an Azure OpenAI endpoint as described [here](../../setup/setup_azure.md).
+
+# %%
+from pyrit.models import PromptRequestPiece, PromptRequestResponse
+from pyrit.prompt_target import AzureOpenAIChatTarget
+from pyrit.common import default_values
+import uuid
+
+default_values.load_default_env()
+test_conversation_id = str(uuid.uuid4())
+
+request_pieces = [PromptRequestPiece(
+    role="user",
+    conversation_id=test_conversation_id,
+    original_prompt_text="Describe this picture:",
+    original_prompt_data_type="text",
+    prompt_target_identifier=None
+), PromptRequestPiece(
+    role="user",
+    conversation_id=test_conversation_id,
+    original_prompt_text="https://t4.ftcdn.net/jpg/00/71/31/85/360_F_71318503_9g1UATIOZzwELZg2DZb7w4oaVyenhQkh.jpg",
+    original_prompt_data_type="image_path",
+    prompt_target_identifier=None
+)]
```

Comment:
```suggestion
request_pieces = [
    PromptRequestPiece(
        role="user",
        conversation_id=test_conversation_id,
        original_prompt_text="Describe this picture:",
        original_prompt_data_type="text",
        prompt_target_identifier=None
    ),
    PromptRequestPiece(
        role="user",
        conversation_id=test_conversation_id,
       
 original_prompt_text="https://t4.ftcdn.net/jpg/00/71/31/85/360_F_71318503_9g1UATIOZzwELZg2DZb7w4oaVyenhQkh.jpg",
        original_prompt_data_type="image_path",
        prompt_target_identifier=None
)]
```
super nit since it looks a bit odd in the current state.

Reply:
This is outdated but it's a good suggestion, followed in another notebook.